feature-libs/product/configurators/rulebased/components/show-more/configurator-show-more.component.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Input,
+  ViewChild,
+} from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  selector: 'cx-configurator-show-more',
+  templateUrl: './configurator-show-more.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfiguratorShowMoreComponent implements AfterViewInit {
+  showMore$ = new BehaviorSubject<boolean>(false);
+  showHiddenText$ = new BehaviorSubject<boolean>(false);
+  textToShow: string;
+
+  @Input() text: string;
+  @Input() textSize = 70;
+  @ViewChild('textContent') textContentElement: ElementRef;
+
+  constructor(private cdRef: ChangeDetectorRef) {}
```

Comment:
use `protected` for extensibility reasons

Reply:
changed to protected